### PR TITLE
feat: add on this page on mobile

### LINKED ISSUE #68: Create "On this page" on mobile for blog
You can use a similar design a vitepress.

### PR DESCRIPTION
**Pull request by Barbapapazes**
### 🔗 Linked issue

fix #68

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Add "on this page" for the mobile version of the blog.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Barbapapazes** [comment]
Missing bg white on button

File: nuxt.config.ts [hunk 1]
```diff
@@ -20,6 +20,9 @@ export default defineNuxtConfig({
     '@nuxthq/studio',
     '@nuxtjs/plausible',
   ],
+  experimental: {
+    inlineSSRStyles: false, // Avoid CSS reset being applied after CSS
```
Comment (pi0) [inline nuxt.config.ts:24]: /cc @danielroe Do you know any better solution to this?

Comment (danielroe) [reply nuxt.config.ts:24]: yes, we can probably set the order of these styles so they come first.

Comment (Barbapapazes) [reply nuxt.config.ts:24]: Yes or we can simply remove inlined SSR styles since only the reset CSS is inlined, I'm not sure about the benefits.

Comment (danielroe) [reply nuxt.config.ts:24]: would you check if enabling `experimental.headNext` fixes anything here?

Comment (Barbapapazes) [reply nuxt.config.ts:24]: Yes, I could give a try to this. Do I have to only enable it or should I add some options?

Comment (danielroe) [reply nuxt.config.ts:24]: it should be enough to enable it

Comment (Barbapapazes) [reply nuxt.config.ts:24]: It change somehting in dev but when file is generated, nothing changed.

Comment (danielroe) [reply nuxt.config.ts:24]: Okay, let's disable it for now and fix in next Nuxt patch.

Comment (Barbapapazes) [reply nuxt.config.ts:24]: Ok. I will open an issue and provide a reproduction.

**Pull request merged**